static/redirections.js

```diff
@@ -95,7 +95,7 @@
         break;
       case "#!adobedocs/adobeio-events/master/using/marketo-user-audit-data-stream-setup.md":
         redirectPath =
-          prefixPath + "guides/using/marketo-user-audit-data-stream-setup/";
+          prefixPath + "guides/using/marketo/marketo-user-audit-data-stream-setup/";
```

Comment:
We should probably include similar redirects for all of our pages

Reply:
yes I agree that's probably what we're covering here I was just thinking to include it for consistency with what already existed and on the off chance someone took the old base path and attached the new page path to it or something, but that's pretty unlikely so not super important, I can go either way whether you want to include the rest here or not